src/Classes/PartyTab.lua

```diff
@@ -0,0 +1,620 @@
+			end
+		end
+	end
+	self:ProcessControlsInput(inputEvents, viewPort)
+
+	main:DrawBackground(viewPort)
+
+	self:DrawControls(viewPort)
+
+	self.modFlag = (self.lastContentAura ~= self.controls.editAuras.buf 
+			or self.lastContentCurse ~= self.controls.editCurses.buf
+			or self.lastContentEnemyCond ~= self.controls.enemyCond.buf
+			or self.lastContentEnemyMods ~= self.controls.enemyMods.buf
+			or self.lastEnableExportBuffs ~= self.enableExportBuffs)
+end
+
+function PartyTabClass:ParseTags(line, currentModType) -- should parse this correctly instead of string match
+	if not line then
+		return "none", {}
+	end
+	local extraTags = {}
+	local modType = currentModType
+	for line2 in line:gmatch("([^,]*),?") do
+		if line2:find("type=GlobalEffect/") then
+			if line2:find("type=GlobalEffect/effectType=AuraDebuff") then
+				t_insert(extraTags, {
+					type = "GlobalEffect",
+					effectType = "AuraDebuff"
+				})
+				modType = "AuraDebuff"
+			elseif line2:find("type=GlobalEffect/effectType=Aura") then
+				t_insert(extraTags, {
+					type = "GlobalEffect",
+					effectType = "Aura"
+				})
+				modType = "Aura"
+			elseif line2:find("type=GlobalEffect/effectType=Curse") then
+				t_insert(extraTags, {
+					type = "GlobalEffect",
+					effectType = "Curse"
+				})
+				modType = "Curse"
+			end
+		elseif line2:find("type=Condition/") then
+			if line2:find("type=Condition/neg=true/var=RareOrUnique") then
+				t_insert(extraTags, {
+					type = "Condition",
+					neg = true,
+					var = "RareOrUnique"
+				})
+			elseif line2:find("type=Condition/var=RareOrUnique") then
+				t_insert(extraTags, {
+					type = "Condition",
+					var = "RareOrUnique"
+				})
+			end
+		end
+	end
+	return modType, extraTags
+end
+
+function PartyTabClass:ParseBuffs(list, buf, buffType)
+	if buffType == "EnemyConditions" then
+		for line in buf:gmatch("([^\n]*)\n?") do
+			list:NewMod(line:gsub("Condition:", "Condition:Party:"), "FLAG", true, "Party")
+		end
+	elseif buffType == "EnemyMods" then
+		local modeName = true
+		local currentName
+		for line in buf:gmatch("([^\n]*)\n?") do
+			if modeName then
+				currentName = line
+				modeName = false
+			else
+				modeName = true
+				local modStrings = {}
+				local modType = currentModType
+				for line2 in line:gmatch("([^|]*)|?") do
+					t_insert(modStrings, line2)
+				end
+				local tags = nil -- modStrings[7], should be done with a modified version of "PartyTabClass:ParseTags" where conditions check vs the party and check that the ones in the build are NOT true, such that your effects override the supports
+				list:NewMod(modStrings[3], modStrings[4], tonumber(modStrings[1]), "Party"..modStrings[2], ModFlag[modStrings[5]] or 0, KeywordFlag[modStrings[6]] or 0, tags)
+			end
+		end
+	end
+	local mode = "Name"
+	if buffType == "Curse" then
+		mode = "CurseLimit"
+	end
+	local currentName
+	local currentEffect
+	local isMark
+	local currentModType = "Unknown"
+	for line in buf:gmatch("([^\n]*)\n?") do
```

Comment:
This format is the same as the dev alt text format, right?  Would be nice to document it

Reply:
yes its based heavily off modlib functions but modified slightly to be more human readable as well as savable into xml, I can document this but unsure where, was planning on having a sort of documentation along with the help file